projects/mci/inc/motor_controller.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "gpio.h"
+
+#include "precharge_control.h"
+
+typedef enum {
+  MCI_CAN_EVENT_RX = 0,
```

Comment:
please put your can event in an `mci_events` file, or `motorcontroller_events.h` file.